app/views/monthlyContributions.scala.html

```diff
@@ -33,9 +34,13 @@
         };
         window.guardian.payPalType = @payPalButton? 'ExpressCheckout' : 'NotSet';
         window.guardian.directDebitEnable = @directDebitButtonEnable;
-        window.guardian.stripeKey = {
-          default: "@defaultStripeConfig.publicKey",
-          uat: "@uatStripeConfig.publicKey"
+        window.guardian.stripeKeyRestOfTheWorld = {
```

Comment:
This might be a little confusing because we typically refer to Rest of World as everywhere except UK/US/Australia. However, as I understand it this key includes UK/US?

Reply:
Yes, in Stripe world we only have two (actually four) set of keys, one for Australia and one for the rest of the world.

Reply:
So if that's the case, would it be best to call this something else? I'm concerned that someone will read this in the future and think it means Guardian RestOfWorld (i.e. everywhere outside of UK/US/Australia), whereas it actually means everywhere but Australia.

Reply:
👍 to @Ap0c's suggestion - I would avoid using the (already overloaded) phrase 'restOfWorld' here, if possible.

Reply:
To be honest, I gave this name because I was not able to come up with something more meaningful. I renamed now to `stripeKeyDefaultCurrencies`, the only problem with this name is that inside there are two environments: test and default. Nevertheless, I feel is less ambiguous.